renderdoc/driver/shaders/spirv/spirv_gen.cpp

```diff
@@ -3338,8 +3332,6 @@ void OpDecoder::ForEachID(const ConstIter &it, const std::function<void(Id,bool)
       callback(Id::fromWord(it.word(3)), false);
       break;
     case rdcspv::Op::Switch:
-      callback(Id::fromWord(it.word(1)), false);
-      callback(Id::fromWord(it.word(2)), false);
```

Comment:
I think there's only currently one case where this would be relevant but we should ensure that this function still iterates over the IDs referenced. I could see this either being done by having a manual implementation of `OpDecoder::ForEachID` which then calls into the autogenerated one (suitably renamed), or just hardcode the handling of Op::Switch in the generator python script to call out to a manually implemented version specifically.